docker/attestation-container/Dockerfile

```diff
@@ -23,7 +23,7 @@ WORKDIR /usr/src/app
 RUN pushd cmd/attestation-container && CGO_ENABLED=0 go test -c && popd && pushd pkg/attest && CGO_ENABLED=0 go test -c && popd && pushd pkg/common && CGO_ENABLED=0 go test -c
 
 # Run
-FROM mcr.microsoft.com/cbl-mariner/distroless/minimal:2.0 AS app-run
+FROM mcr.microsoft.com/cbl-mariner/base/core:2.0 AS app-run
```

Comment:
I thought this was causing issues and we were sticking to alpine for now?

Reply:
it didn't cause issues in this container (still passes the pipeline)